CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+# Contributing to a Rearc AWS Data Exchange product
+
+🎉🥳 First of all, **THANK YOU** for being interested in contributing to one of our projects 🎉🥳
+
+From the start of our AWS Data Exchange (ADX) initiative we knew we wanted to have an open dialogue with our subscribers. While we try our best to optimize the packaging of our ADX products to be as accommodating as possible, we appreciate there may be use-cases we have overlooked. By contributing to this project, you have an opportunity to improve the usability of the data included with this product not only for yourself, but for anyone who subscribes to this product through ADX. If your contribution is applicable to more than one of our ADX listings, we will gladly improve those other products as well!
+
+You can learn about Rearc’s data services by visiting [https://www.rearc.io/data]( https://www.rearc.io/data).
+
+Below is a set of guidelines for contributing to any of the ADX product repositories available on the [Rearc Data GitHub account](https://github.com/rearc-data).
+
+#### Table of Contents
+- [I have a question and/or issue to report. What is the best way to contact Rearc?](#i-have-a-question-andor-issue-to-report-what-is-the-best-way-to-contact-rearc)
+- [What should I know before getting started?](#what-should-i-know-before-getting-started)
+  * [What is AWS Data Exchange?](#what-is-aws-data-exchange)
+  * [Who is Rearc?](#who-is-rearc)
+  * [What are Rearc's goals for ADX?](#what-are-rearcs-goals-for-adx)
+  * [What is Rearc's philosophy towards dataset formats?](#what-is-rearcs-philosophy-towards-dataset-formats)
+  * [What tools are you using throughout your ADX Products?](#what-tools-are-you-using-throughout-your-adx-products)
+  * [Directory Layout](#directory-layout)
+- [How can I contribute?](#how-can-i-contribute)
+  * [Report an Issue/Bug](#report-an-issuebug)
+  * [Submit an Improvement/Suggestion](#submit-an-improvementsuggestion)
+  * [Pull Request](#pull-request)
+- [Additional Resources](#additional-resources)
+
+## I have a question and/or issue to report. What is the best way to contact Rearc?
+If you have an issue to report specific to the source code in this repository you can [open a GitHub issue](https://github.com/rearc-data/publish-a-data-product-on-aws-data-exchange/issues). Before opening an issue please review the existing suggestions to see if your idea is already there. If already present, please comment on the existing issue instead of making a new one.
```

Comment:
Looks good but including a link might make it hard to sync this file between all our rearc-data repositories.

Reply:
Removed the section on line 26 and 27, and incorporated parts of it into the "Report an Issue/Bug or Submit an Improvement/Suggestion" section further down in the document